### PR TITLE
[search] Use smaller lookupRadiusM for postcodes builder.

### PR DESCRIPTION
**Pull request by tatiana-yan**

### CODE REVIEW

**Comment by tatiana-yan** [comment]
JTLINUX

File: generator/search_index_builder.cpp [hunk 1]
```diff
@@ -277,7 +277,7 @@ void GetUKPostcodes(string const & filename, storage::CountryId const & countryI
     auto const p = MercatorBounds::FromLatLon(lat, lon);
 
     vector<storage::CountryId> countries;
-    infoGetter.GetRegionsCountryId(p, countries);
+    infoGetter.GetRegionsCountryId(p, countries, 200 /* lookupRadiusM */);
```
Comment (gmoryes) [inline generator/search_index_builder.cpp]: 200.0

File: storage/country_info_getter.hpp [hunk 2]
```diff
@@ -73,7 +73,8 @@ class CountryInfoGetter : public CountryInfoGetterBase
   // |closestCoutryIds| is filled with country ids of mwms that cover |pt| or are close to it
   // with the exception of World.mwm and custom user-provided mwms.
   // The result may be empty, for example if |pt| is somewhere in an ocean.
-  void GetRegionsCountryId(m2::PointD const & pt, CountriesVec & closestCoutryIds);
+  void GetRegionsCountryId(m2::PointD const & pt, CountriesVec & closestCoutryIds,
+                           double lookupRadiusM = 30 /* km */ * 1000);
```
Comment (gmoryes) [inline storage/country_info_getter.hpp]: .0 too

**Comment by tatiana-yan** [comment]
JTLINUX

File: generator/search_index_builder.cpp (hunk 1 above)
Comment (tatiana-yan) [reply generator/search_index_builder.cpp]: done

File: storage/country_info_getter.hpp (hunk 2 above)
Comment (tatiana-yan) [reply storage/country_info_getter.hpp]: done

**Pull request merged**